pcm_getStartingval.m

```diff
@@ -23,5 +23,7 @@
         error('not implemented yet'); 
     case 'nonlinear' 
         error('cannot provide starting values for nonlinear models'); 
+    case 'noiseceiling'
```

Comment:
For getStartingval, noise ceiling models shouldn't require a starting value (per se), so this case was added to avoid errors.